### PR TITLE
fix(content): STRF-4948 fix missing svg icons on AMP add-to-cart page

### PR DESCRIPTION
**Pull request by davidwrpayne**
#### What?
fix missing svg icons on AMP add-to-cart page by generating an template file that contains the assets and referencing that as before from Cornerstone 2.0.0 changes.

#### Tickets / Documentation
- [STRF-4848](https://jira.bigcommerce.com/browse/STRF-4948)

#### Screenshots (if appropriate)
screen shot 2018-06-07 at 12 28 26 pm

@bigcommerce/storefront-team @mattolson

### CODE REVIEW

File: assets/img/icon-sprite.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...8.8527593 C31.6634051,8.8527593 31.3509198,6.64879843 30.3919217,5.67824658 C29.1757339,4.40441487 27.8125088,4.39809002 27.1873503,4.32353816 C22.7118278,4 15.9983092,4 15.9983092,4 L15.984407,4 C15.984407,4 9.27104501,4 4.79536595,4.32353816 C4.17017613,4.39809002 2.80745205,4.40441487 1.59082583,5.67824658 C0.631890411,6.64879843 0.319843444,8.8527593 0.319843444,8.8527593 C0.319843444,8.8527593 0,11.4409393 0,14.0290881 L0,16.4554834 C0,19.0436008 0.319843444,21.6317495 0.319843444,21.6317495 C0.319843444,21.6317495 0.631890411,23.8357417 1.59082583,24.8062935 C2.80745205,26.0801566 4.40557339,26.0398591 5.11736986,26.1733699 C7.67602348,26.4187241 15.9913894,26.4946536 15.9913894,26.4946536 C15.9913894,26.4946536 22.7118278,26.4845401 27.1873503,26.1610333 C27.8125088,26.0864501 29.1757339,26.0801566 30.3919217,24.8062935 C31.3509198,23.8357417 31.6634051,21.6317495 31.6634051,21.6317495 C31.6634051,21.6317495 31.9827789,19.0436008 31.9827789,16.4554834 L31.9827789,14.0290881 C31.9827789,11.4409393 31.6634051,8.8527593 31.6634051,8.8527593 Z M12.6895342,19.39582 L12.6880626,10.4095186 L21.3299413,14.9183249 L12.6895342,19.39582 Z" id="youtube-Imported-Layers"/> </symbol></svg>
```
Comment (mattolson) [inline assets/img/icon-sprite.svg]: This file shouldn't have a diff. I believe this is related to https://github.com/bigcommerce/cornerstone/issues/1246. We need to remove the "mask" and " sketch:type" attributes from instagram.svg and stumbleupon.svg. These are included in the svg sprite and causing issues when loading in certain browsers. I believe there were some manual changes to the sprite in an earlier release.

Comment (davidwrpayne) [reply assets/img/icon-sprite.svg]: Ah I see, I'll see what I can do.

File: assets/icons/stumbleupon.svg [hunk 2]
```diff
@@ -6,5 +6,5 @@
     <mask id="mask-2" sketch:name="Clip 2">
         <use xlink:href="#path-1"></use>
     </mask>
```
Comment (davidwrpayne) [inline assets/icons/stumbleupon.svg]: @mattolson  Do I need to also remove this tag? I only removed the attribute from the path tag below.

Comment (mattolson) [reply assets/icons/stumbleupon.svg]: No I think this one is fine.

Comment (mattolson) [reply assets/icons/stumbleupon.svg]: Yes, you need to remove `sketch:name` from the mask.

Comment (mattolson) [reply assets/icons/stumbleupon.svg]: You also need to remove the `xlink` prefix from the `use` tag: `<use href=...`

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,10 @@
 
 ## Draft
 - Fix quantity edit on Simple Product AMP pages. [#1257](https://github.com/bigcommerce/cornerstone/pull/1257)
+<<<<<<< HEAD
```
Comment (davidwrpayne) [inline CHANGELOG.md]: take that out

**Comment by davidwrpayne** [comment]
@mattolson Validated that the icons work on a store.

**Pull request merged**